nemo/collections/multimodal/speech_llm/models/modular_models.py

```diff
@@ -1561,3 +1475,75 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
         )
         results.append(model)
         return results
+
+
```

Comment:
Maybe put in another file, otherwise as we start getting more variants this file will explode in size

Reply:
Acknowledged. Try to put all similar gpt based speech_llm in modular_models.py for now. We can split to different categories when this list grows